resources/js/components/GameRow.js

```diff
@@ -2,7 +2,8 @@
 
 import React, { useState } from "react";
 import PlayerList from './PlayerList';
-import {IoIosArrowDown, IoIosArrowUp} from 'react-icons/io'
+import {IoIosArrowDown, IoIosArrowUp} from 'react-icons/io';
+import './GameRow.css';
 
 export default function GameRow(data) {
```

Comment:
you could destructure things like this `({game})`. Then you wouldn't always have to do `data.game`